mappings/net/minecraft/item/ItemStack.mapping

```diff
@@ -77,6 +77,9 @@ CLASS bca net/minecraft/item/ItemStack
 		ARG 2 modifier
 	METHOD a setChildTag (Ljava/lang/String;Lis;)V
 		ARG 1 tagName
+	METHOD a appendEnchantmentComponents (Ljava/util/List;Lii;)V
+		ARG 0 list
```

Comment:
What kind of list? `components`, going from the method's name?

Reply:
probablu adding from a list of info enchantment to a list tag

Reply:
In that case I'd prefer `tag` or `nbt`

Reply:
It's the list the method should append to. Naming it `components` may make it seem like they are the components being appended, while `list` makes it clear it's an output parameter.